spec/coffeescripts/views/conferences/ConferenceViewSpec.js

```diff
@@ -109,11 +109,22 @@ test('deleteRecordings calls screenreader', function() {
     id: 1,
     recordings: [
       {
-        duration_minutes: 0,
-        ended_at: 1518554650000,
-        playback_url: "www.blah.com",
```

Comment:
I tend to think it's best to stick with "example.com" in tests like this, but pretty minor.

Reply:
Fair enough, no reason to do that in this commit.